tagstudio/src/qt/widgets/video_player.py

```diff
@@ -51,6 +49,25 @@ class VideoPlayer(QGraphicsView):
     content_visible = False
     filepath = None
 
+    pause_icon: bytes = None
+    play_icon: bytes = None
+    volume_mute_icon: bytes = None
+    volume_icon: bytes = None
+
+    # Load icon files as bytes
+    _parents = Path(__file__).parents[3]
+    with open(Path(_parents, "resources/qt/images/pause.svg"), "rb") as icon:
+        pause_icon = bytes(icon.read())
+
+    with open(Path(_parents, "resources/qt/images/play.svg"), "rb") as icon:
+        play_icon = bytes(icon.read())
+
+    with open(Path(_parents, "resources/qt/images/volume.svg"), "rb") as icon:
+        volume_icon = bytes(icon.read())
+
+    with open(Path(_parents, "resources/qt/images/volume_mute.svg"), "rb") as icon:
+        volume_mute_icon = bytes(icon.read())
```

Comment:
I'd keep these resources outside of the `VideoPlayer` class, and rather have another class for handling this kind of assets, `ResourceManager` or something like that. Then it can be used across the whole codebase for all the resources, which would be nicer than doing `open(Path(_parents, "resources/qt/something/something`))` everywhere. 
This will provide some level of abstraction, so if/when the underlying implementation will change, it wont affect any code using the resources.

Something like this:

```python
class ResourceManager:
    MAP = {
        "play_icon": "qt/images/play.svg",
        # ...
    }

    def _load(self, id: str):
        # eventually implement cache like
        # if id in self.CACHE: return self.CACHE[id]
        with open(self.MAP[id], 'rb') as f:
            return f.read()

    def get(self, id: str):
        """Allow to use the manager as:
        r = ResourceManager()
        r.get('play_icon')
        """
        return self._load(id)

    def __getattr__(self, id: str):
        """Allow to use the manager as:
        r = ResourceManager()
        r.play_icon
        """
        if id in self.MAP:
            return self._load(id)
        raise AttributeError(f'attribute {id} not found')
```